src/uwtools/tests/config/test_jinja2.py

```diff
@@ -133,14 +130,6 @@ def test_dereference_str_filter_rendered():
     assert jinja2.dereference(val=val, context={"recipient": "world"}) == "hello, world"
 
 
-def test_dereference_str_variable_rendered_int():
-    # Due to reification, the value of a result parsable as an int is an int. The same holds for
-    # other results parsable by YAML as Python values, but this is only a representative, non-
-    # exhaustive test.
-    val = "{{ number }}"
-    assert jinja2.dereference(val=val, context={"number": "88"}) == 88
```

Comment:
The removed tests here and above are handled better by newly added tests, below.